docs/sdk/python.md

```diff
@@ -132,23 +132,23 @@ The delayed message is non-blocking so functions will continue to process record
 The delay is specified via a [Python timedelta](https://docs.python.org/3/library/datetime.html#datetime.timedelta).
 
 {% highlight python %}
-from google.protobuf.any_pb2 import Any
+from datetime import timedelta
 from statefun import StatefulFunctions
 
 functions = StatefulFunctions()
 
-@functions.bind("example/caller")
-def caller_function(context, message):
-    """A simple stateful function that sends a message to the user with id `user1`"""
+@functions.bind("example/delay")
+def delayed_function(context, message):
+    """A simple stateful function that sends a message to its caller with a delay"""
 
-    user = User()
-    user.user_id = "user1"
-    user.name = "Seth"
+    Response response = Response()
```

Comment:
this seems like a typo. perhaps you meant: `response = Response()` ?

Reply:
This is what happens when I write Java all day and then try to quickly fix some python